.github/ISSUE_TEMPLATE/hotfix.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
@foxleigh81 - I think we could do with explaining the difference between bug report & hotfix (I don't know...)

Reply:
You know what, technically a hotfix is for a release which we don't really have a need for. So I'm going to rethink the bug and hotfix thing a bit.

Reply:
I've replaced `hotfix` with `amend request` I used hotfix as I'm used to that term but it doesn't technically apply in this context as we don't really have to work on 'released' code we just do work for a next release. 

So now the difference is:

> An amend is different to a bug. A bug is an issue that causes a failure or error somewhere, whereas an amend is something that isn't quite right. For example a page not loading is a bug, a page having a header that needs to be bigger is an amend. 

This information is also in the development standards document :)